metpy/plots/skewt.py

```diff
@@ -770,10 +771,70 @@ def plot(self, u, v, **kwargs):
         u, v = delete_masked_points(u, v)
         return self.ax.plot(u, v, **line_args)
 
-    def plot_colormapped(self, u, v, c, **kwargs):
+    def plot_layers(self, u, v, hgt, bounds, colors, **kwargs):
```

Comment:
Is this function sticking around after the functionality was implanted in `plot_colormapped`?